src/modules/item/utils.ts

```diff
@@ -31,6 +31,10 @@ export function getMaxSupply(item: Item) {
   return RARITY_MAX_SUPPLY[item.rarity!]
 }
 
+export function getMaxSupplyForRarity(rarity: ItemRarity) {
```

Comment:
This method is great! but I think we can refactor [one above](https://github.com/decentraland/builder/pull/1527/files#diff-6e44a8ab5281e24e1251f5bee8340d324289f9b25dfbf8f77aba93077b4c71b5R30) to avoid repetition right?

Reply:
done!